File: yizazunapi/src/main/java/com/advancedmobileprogramming/yizazunapi/models/Food.java
package com.advancedmobileprogramming.yizazunapi.models;

import lombok.Data;

import javax.persistence.*;
import java.util.List;

@Data
@Entity
public class Food {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String food_name;

    private String food_description;

    private int price;
    
    @Lob
    private byte[] food_image;

}

File: Yizazun/app/src/main/java/com/advancedmobileprogramming/yizazun/model/Food.kt
package com.advancedmobileprogramming.yizazun.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import java.io.Serializable

@Entity (tableName = "food")
class Food (
    @PrimaryKey(autoGenerate = true) @ColumnInfo(name = "id") val id:Long,
    @ColumnInfo(name = "food_name") val food_name:String,
    @ColumnInfo(name = "food_description") val description:String,
    @ColumnInfo(name = "price") val price:Int,
    @ColumnInfo(typeAffinity = ColumnInfo.BLOB,name="food_image") var food_image: ByteArray
):Serializable